src/frontend/lib/utils.ts

```diff
@@ -114,17 +114,11 @@ export function filterPhotosFromAttachments(
     )
       acc.push({ id: att.id, type: att.type });
     return acc;
-  }, []);
-}
-
-export function getLastPhotoAttachment(
-  attachments?: Array<ObservationAttachment> = []
-): SavedPhoto | void {
-  return filterPhotosFromAttachments(attachments).pop();
```

Comment:
Did you get rid of this on purpose?

Reply:
yes this isn't being used anywhere so figured it would make sense to remove